kong/runloop/handler.lua

```diff
@@ -992,7 +952,7 @@ return {
         -----------------------------------------------------------------------
         -- Serviceless stream route
         -----------------------------------------------------------------------
-        local service_scheme = ssl_termination_ctx and "tls" or "tcp"
+        local service_scheme = "tcp"
```

Comment:
I am not sure about this. Couldn't the scheme still be `tls`?

Reply:
I am also not sure about this one. pinging @dndx

Reply:
TLS is not supported until the `stream_listen` `ssl` flag thing is added.